src/config_types.ts

```diff
@@ -101,7 +101,7 @@ function contextIterator(): u.ListIterator<any, Context> {
         if (!elt.context.cluster) {
             throw new Error(`contexts[${i}].context.cluster is missing`);
         }
-        if (!elt.context.user) {
+        if (elt.context.user === undefined) {
```

Comment:
Not quite sure of the import of this change.  Is the intention to permit the empty string (or `null`), but still forbid `undefined`?

Reply:
That's what I was going for. TBH, I don't know what the "right" expectation is here.... I can test with a config with no user at all and see what `kubectl` does...

Reply:
Looks like kubectl is ok with it 100% missing, so I just removed this.